src/platform/guigtk.cpp

```diff
@@ -576,9 +578,11 @@ class GtkGLWidget : public Gtk::GLArea {
         gdk_event_get_scroll_deltas((GdkEvent*)gdk_event, &dx, &dy);
         
         double delta;
-        if(dy < 0 || dir == GDK_SCROLL_UP) {
+        if(abs(dy) > 0) {
```

Comment:
An interesting result from superficial googling that may be relevant :-)
https://stackoverflow.com/questions/11775161/gtk3-get-mouse-scroll-direction

Reply:
@pontaoski I made an alternative implementation based on the above suggestion here:
https://github.com/solvespace/solvespace/pull/1470
Since my only Linux system is a RaspberryPi I have neither Wayland nor a trackpad. `gdk_event_get_scroll_deltas` does return true and `dy` is always -1 or 1.

What do you think, which approach is better?